titan-server/src/routes.rs

```diff
@@ -83,4 +87,73 @@ pub fn woltlab_login(
 #[get("/pulse", format = "application/json")]
 pub fn health_check(token: auth_guard::AuthenticatedUser) -> Json<auth_guard::AuthenticatedUser> {
     return Json(token)
+}
+
+/** **************************************************
+ *  Roster
+ ** **************************************************/
+#[derive(Serialize)]
+pub struct RosterMember {
+    user: models::WcfUser,
+    avatar: String
+}
+
+#[derive(Serialize)]
+pub struct FindBranchResponse {
+    branch: models::TitanBranch,
+    roster: Vec<RosterMember>
+}
+
+#[get("/<branch_slug>")]
```

Comment:
I agree with the structure, though I'd like to rebrand "branch" to a more general entity: "organization". In our community, we have branches, as well squads (for games with less members), intel groups, administrative roles, and occasionally competitive teams. All of these entities are aliases for the same core information.

The below schema gives us the necessary flexibility to support many types of organizations in our community with minimal changes:

```
| id | name        | slug        | type     | is_enabled | wcf_user_group_id |
|----|-------------|-------------|----------|------------|-------------------|
| 1  | Air Force   | air-force   | division | 1          | 32                |
| 2  | Engineering | engineering | intel    | 1          | 63                |
```

1. Add a `type` field: Identifies which parent group the org belongs to. In the future (and only if necessary) we can eventually swap it out with a dedicated entity to allow for more complex relationships.
2. Rename references of `branch` to `organization: If we're going to be moving code and renaming files anyway, we might as well just handle this now.

The UI would remain the same, where users can view each branch at `/branch/{org slug}`. Thoughts?

Reply:
I think is a well-reasoned approach. It doesn't put the cart before the horse. It just plans for a more complex cart as we build things out. I like it.

I think we definitely will swap that `type` field out in the future, I just don't have exact timing on when that future arrives.

I think point #2 makes sense given what we're doing now.

Let's do it 👍

Reply:
It turns out the script in #2 isn't the most efficient way to achieve that behavior. We can create path aliases in the webpack config instead. In fact, the app is already referencing files from `titan/...` to avoid a mess of relative paths.